components/bpmn/org.wso2.carbon.bpmn/src/main/java/org/wso2/carbon/bpmn/core/ActivitiEngineBuilder.java

```diff
@@ -53,84 +53,107 @@
 import org.wso2.carbon.bpmn.core.types.datatypes.json.JsonAPIResolverFactory;
 import org.wso2.carbon.bpmn.core.types.datatypes.xml.XmlAPIResolverFactory;
 import org.wso2.carbon.bpmn.core.types.datatypes.xml.XmlType;
+import org.wso2.carbon.bpmn.core.utils.SecretVaultValueReader;
 import org.wso2.carbon.utils.CarbonUtils;
 
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
+import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  * Class responsible for building and initiating the activiti engine
- *
  */
 public class ActivitiEngineBuilder {
 
-	private static final Log log = LogFactory.getLog(ActivitiEngineBuilder.class);
-
-	private String dataSourceJndiName = null;
+    private static final Log log = LogFactory.getLog(ActivitiEngineBuilder.class);
     private static ProcessEngine processEngine = null;
     protected ObjectMapper objectMapper = new ObjectMapper();//Object mapper for JsonType,ExtendedJsonType,LongJsonType
+    private String dataSourceJndiName = null;
+
+    /* Instantiates the engine. Builds the state of the engine
+     *
+     * @return  ProcessEngineImpl object
+     * @throws BPSFault  Throws in the event of failure of ProcessEngine
+     */
 
```

Comment:
Shall we remove the empty new line between the comment and the method declration?

Reply:
Done